.gitignore

```
@@ -102,3 +102,7 @@ dist
 
 # TernJS port file
 .tern-port
+
+.python_packages
+__pycache__
+.venv
```

Just click the lil commit button 
```suggestion
.venv
.vscode/
```